### ISSUE TITLE
Torrents for download images

### ISSUE THREAD

**Issue by goetzk** [maemo-leste/bugtracker#63]
Hi,
With the AMD64 VMs from #45 there are now some very large >1.5GB) files to download.
It might help with distribution to provide torrents, at least there will be the possibility for people to share the load.

thanks,

**Comment by MerlijnWajer** [comment]
Good idea. Shall we do that once we make a beta release or so? I think we're a few releases (weeks, months...) away for releases for "users".

**Comment by parazyd** [comment]
Let's avoid torrents while we're building images often. Once things are
more stable, then it should be okay to provide.

**Comment by goetzk** [comment]
On 8/2/18 16:53, Merlijn Wajer wrote:
> Good idea. Shall we do that once we make a beta release or so? I think
> we're a few releases (weeks, months...) away for releases for "users".

I suggest for anything that you expect to see distributed widely.
Betas/official releases make the most sense, but if you are pushing a
particular snapshot to a group of people it might make sense there too.

kk
